MAINTAINERS

```diff
@@ -77,6 +78,11 @@
 	Email = "mazz@houseofmnowster.com"
 	GitHub = "mnowster"
 
+	[people.ndeloof]
+	Name = "Nicolas De Loof"
+	Email = "nicolas.deloof@docker.com"
```

Comment:
Please use nicolas.deloof@gmail.com for everything OSS

Reply:
ok